CHANGELOG.md

```diff
@@ -4,9 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.12.0] - 2024-01-10
+### Added
+- `ChatMessageInfo.ChatFeedbackInfo` to include Explorer feedback (#107)
+- `Message.messageId` (#107)
+- `PlatformClient.sendMessageFeedback` to send user feedback to the AI response (#107)
+
 ## [1.11.0] - 2024-01-03
 ### Added
-- `User.aiDailyMessageLimit` to determine how many AI image description requests an Explorer can make per day (#102)
+- `User.aiDailyMessageLimit` to determine how many AI image description requests an Explorer can make per day (#106)
```

Comment:
Why change this from #102?

Reply:
No, but the number was't correct in the previous commit, so I have changed to prevent future misunderstanding